llvm/lib/Analysis/BasicAliasAnalysis.cpp

```diff
@@ -594,9 +580,9 @@ BasicAAResult::DecomposeGEPExpression(const Value *V, const DataLayout &DL,
   SearchTimes++;
   const Instruction *CxtI = dyn_cast<Instruction>(V);
 
-  unsigned MaxIndexSize = DL.getMaxIndexSizeInBits();
```

Comment:
looks like `getMaxIndexSizeInBits()` is only used here

Reply:
Dropped the API in https://github.com/llvm/llvm-project/commit/07aab4a3cdab3d46caab270845413c5ba4546b50.